cmd/fyne/internal/mobile/build_iosapp.go

```diff
@@ -229,6 +232,8 @@ func iosCopyAssets(pkg *packages.Package, xcodeProjDir string) error {
 type infoplistTmplData struct {
 	BundleID string
 	Name     string
+	Version  string
+	Build    int
 }
 
 var infoplistTmpl = template.Must(template.New("infoplist").Parse(`<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this perhaps be moved out to a separate file in the internal template package?